changelogs/fragments/9694-ipa-host-certificate-revoked.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - host_add - fix. The module revoked existing host certificates even if ``user_certificates`` was not given. (https://github.com/ansible-collections/community.general/pull/9694).
```

```suggestion
  - ipa_host - module revoked existing host certificates even if ``user_certificate`` was not given. (https://github.com/ansible-collections/community.general/pull/9694).
```

Please refer to the standard formatting for the changelog fragments' entries:

https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to-format

The outcome of what you write will be published in the collection changelog:

https://docs.ansible.com/ansible/latest/collections/community/general/changelog.html